tests/haskell-load-tests.el

```diff
@@ -0,0 +1,82 @@
+;;; haskell-load-tests.el
+
+;;; Code:
+
+(require 'ert)
+(require 'haskell-test-utils)
+
+(require 'haskell-load)
+
+(defun insert-errors ()
+  (insert "import           Control.Applicativ\nimport           Data.Mayb\nimport           Data.String")
+  (let ((a (make-overlay 18 36))
```

Comment:
Please use `re-search-forward` and then `(match-begin 0)` and `(match-end 0)` to put the overlays. Should somebody come and need to change these tests they will be intimidated by character counting and won't do it.